client/my-sites/plans/jetpack-plans/product-lightbox/payment-plan.tsx

```diff
@@ -19,23 +19,33 @@ type PaymentPlanProps = {
 	product: SelectorProduct;
 	showPlansOneBelowTheOther?: boolean;
 	isActive?: boolean;
+	quantity?: number | null;
 };
 const PaymentPlan: React.FC< PaymentPlanProps > = ( {
 	isMultiSiteIncompatible,
 	siteId,
 	product,
 	showPlansOneBelowTheOther = false,
 	isActive = true,
+	quantity = null,
 } ) => {
 	const translate = useTranslate();
 	const { containerRef, isCompact } = useItemPriceCompact();
 
-	const { originalPrice, discountedPrice, discountedPriceDuration, isFetching } = useItemPrice(
-		siteId,
-		product,
-		product?.monthlyProductSlug || ''
-	);
+	const { originalPrice, discountedPrice, discountedPriceDuration, isFetching, priceTierList } =
+		useItemPrice( siteId, product, product?.monthlyProductSlug || '' );
+
+	const currentTier =
+		quantity &&
+		priceTierList.find( ( tier ) => {
+			if ( tier.maximum_units ) {
+				return quantity >= tier.minimum_units && quantity <= tier.maximum_units;
```

Comment:
Ah, I did not consider - if we check against both min an max units here - do we also need to verify that `tier.minimum_units` exists too?

Reply:
In the types, minimum_units is labelled as a required value. Because of that, I don't think we need to?